GeometryService/inc/STMMaker.hh

```diff
@@ -102,6 +112,32 @@ namespace mu2e {
     double       _pipeFlangeHalfLength;
     double       _pipeFlangeOverhangR;
 
+    bool        _absorberBuild;//wyq
```

Comment:
In general we can remove author comments from lines. Git should generally tell us who added which line of code if needed